environment-dev.yml

```diff
@@ -82,3 +81,4 @@ dependencies:
       - ray
       - PyGithub
       - pypandoc
+      - sherpa
```

Comment:
Why sherpa is a dev dependency, as stated l49?

Reply:
I'm unsure why that file changed, will fix! Thanks for picking up on it